cmd/deploy/cmd.go

```diff
@@ -6,20 +6,29 @@ import (
 
 const (
 	use = "deploy"
-	sho = "Claim the given test environment and continuously deploy the given service branch."
-	lon = "Claim the given test environment and continuously deploy the given service branch."
+	sho = "Manually trigger a CloudFormation stack update."
+	lon = "Manually trigger a CloudFormation stack update."
```

Comment:
The idea here is that the `kayron deploy` command becomes the thing that we use "manually" in the Github Action in our infrastructure repo.

We can implement claiming test environments with another command later. The place holder here was just randomly assigned when the project started out.